### ISSUE TITLE
[FEATURE] Allow connection to mySQL server over SSH

### ISSUE THREAD

**Issue by ghost** [tlayh/vagrant-typo3cms#11]
I could not manage to connect to the mySQL server over SSH from the host machine. is this a missing feature or is there maybe something i did wrong?

these are the connection settings i used:

http://d.pr/i/sDDk

it leads to this error messages:

http://d.pr/i/C5pG
http://d.pr/i/GED8

note: i can successfully connect to the mySQL server in your TYPO3 Neos vagrant box [1]. i already tried to find the differences in the vagrant/chef configuration between these to vagrant boxes but could not manage to find the configuration which could solve my problem.

are you able to connect to the mySQL server over SSH? do you know a better way? is there a solution for my problem?

kind regards

roland

[1] http://d.pr/i/5RON

**Comment by tlayh** [comment]
Are both boxes running at the same time? Make sure the SSH port is the correct one. If vagrant detects that one box is already running it is taking a different SSH port. But I must say I have never tried to connect to the database via SSH.

**Comment by ghost** [comment]
no, i destroyed the TYPO3 Neos box before. thank you for your fast relply anyway!

**Comment by tlayh** [comment]
Go to /etc/mysql and edit my.cnf

Find: bind-address            = 10.0.2.15
Remove the line, restart mysql and it will work. I am not sure how this can be set directly in the setup.

**Comment by tlayh** [comment]
Fix commited, I removed the bind_address parameter for now.

**Comment by tlayh** [comment]
Btw. for some reason I had to use SSH Port 22 instead of 2222

**Comment by tlayh** [comment]
![settings-typo3cms-sequel](https://f.cloud.github.com/assets/285669/1648930/5ebbb432-59cf-11e3-9dc6-ac3464a49f44.jpg)
